apps/explorer/src/pages/validators/Validators.tsx

```diff
@@ -30,13 +30,15 @@ import { roundFloat } from '~/utils/roundFloat';
 
 const APY_DECIMALS = 3;
 
+// This constant needs to match the constant in the on-chain smart contract sui_system::MIN_VALIDATOR_STAKE。
+const MIN_VALIDATOR_STAKE = 25_000_000_000_000_000;
```

Comment:
this should be fine short-term for but ideally we could read this value from somewhere / share a constant